src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/WebRequestPSCmdlet.Common.cs

```diff
@@ -1269,12 +1269,10 @@ internal virtual HttpResponseMessage GetResponse(HttpClient client, HttpRequestM
                     }
 
                     currentUri = new Uri(request.RequestUri, response.Headers.Location);
+
                     // Continue to handle redirection
-                    using (client = GetHttpClient(handleRedirect))
-                    using (HttpRequestMessage redirectRequest = GetRequest(currentUri))
-                    {
-                        response = GetResponse(client, redirectRequest, handleRedirect);
-                    }
+                    using HttpRequestMessage redirectRequest = GetRequest(currentUri);
+                    response = GetResponse(client, redirectRequest, handleRedirect);
```

Comment:
Nit: it would great to inspect how we dispose HttpResponseMessage

Reply:
I noticed that:
- `response = GetResponse(client, redirectRequest, handleRedirect);` line 1275
- `return GetResponse(client, requestWithoutRange, handleRedirect);` line 1309
are doing the same thing, I think they should have similar formatting, which one is better?

To dispose of HttpResponseMessage we could add using on line 1422, and it might be enough

Reply:
I don't understand the question about formatting.

> To dispose of HttpResponseMessage we could add using on line 1422

Yes. And I see two callsites `return GetResponse` - I guess we need to dispose previous response before return new one.

Reply:
I'm sorry, I meant they should have the same syntax (not formatting) . And I wanted to know which syntax was more clear. In the next PR

Reply:
I still don't understand. Feel free to pull PR with your prefrence.